.travis.yml

```diff
@@ -190,7 +208,8 @@ install:
   - cmake .. -DCMAKE_BUILD_TYPE=${BUILD_TYPE}
   - make --jobs=${JOBS}
   - popd
-  - npm run build-api-docs
+  # building docs only works with npm3+ not with yarn or npm2
+  #- yarn run docs
```

Comment:
This needs fixes in the docs tool. I don't see the point of building the docs for every build job. It almost takes a minute and would block us from using a sensible package manager like yarn.